docs/src/user/rpc.md

```diff
@@ -30,9 +30,15 @@ Request proofs for a set of nullifiers.
 
 ## CheckNullifiersByPrefix
```

Comment:
I'm thinking we should rename this to `SyncNullifiers` endpoint. This would be consistent with having all endpoints which may return "chunked" data to have a `Sync` prefix.

Reply:
Should I opened an issue to discuss whether to use `Sync`, `Paged` or another prefix?